tests/openthread_test/Makefile

```diff
@@ -0,0 +1,47 @@
+APPLICATION = openthread_test
```

Comment:
Yes. One can easily switch over the `posix_socket` example by changing the included modules (which I might do in the future anyways ;-))